client/my-sites/customer-home/cards/tasks/use-built-by/index.js

```diff
@@ -0,0 +1,23 @@
+import { useTranslate } from 'i18n-calypso';
+import announcementImage from 'calypso/assets/images/marketplace/built-by-wp-hori-blue.svg';
+import { TASK_USE_BUILT_BY } from 'calypso/my-sites/customer-home/cards/constants';
+import Task from 'calypso/my-sites/customer-home/cards/tasks/task';
+
+const UseBuiltBy = () => {
+	const translate = useTranslate();
+
+	return (
+		<Task
+			title={ translate( 'Hire experts to build your website' ) }
```

Comment:
Check comment I added in D90417-code#1907160. Since "Built By" appears to be only for `en` language users, I suggest that we restrict the view to `en` locale in the backend and remove the `translate()` calls here. If we're only going to show to `en` users, then translating them incurs costs which can be avoided.

Reply:
Fixed.